src/components/sidebar/sidebar.component.tsx

```diff
@@ -122,7 +122,14 @@ export const Sidebar = React.forwardRef<HTMLDivElement, SidebarProps>(
   ) => {
     const locale = useLocale();
     const { current: headerId } = useRef<string>(createGuid());
-    const hasStickyFooter = useIsStickyFooterForm(children);
+    const hasStickyFooter = useMemo(
+      () =>
+        React.Children.toArray(children).some(
+          (child) =>
+            React.isValidElement<FormProps>(child) && child.props.stickyFooter
+        ),
+      [children]
+    );
```

Comment:
This will need updating as well at a future date, since any wrapped `Form`s won't be recognised

Reply:
we could use DOM querying here

``` 
const hasStickyFooter = () => {
  const form = ref.current?.querySelector("[data-component='form']");
  
  return form.classList.contains("sticky");
};